### ISSUE TITLE
Kibana replies with 400

### ISSUE THREAD

**Issue by faust64** [chamilad/kibana-prometheus-exporter#7]
As I could not find any Kibana metrics in my prometheus, I realized the exporter fails to gather its metrics:

```
$ kubectl logs -n logging kibana-kube-5d57f6fd4c-64rpt -c exporter
2020/11/22 17:19:34 using Kibana URL: http://localhost:5601
2020/11/22 17:19:34 Kibana username or password is not provided, assuming unauthenticated communication
2020/11/22 17:19:34 starting metrics server at :9684
2020/11/22 17:20:54 error while scraping metrics from Kibana: invalid response from Kibana status: 400 Bad Request
2020/11/22 17:21:24 error while scraping metrics from Kibana: invalid response from Kibana status: 400 Bad Request
2020/11/22 17:21:54 error while scraping metrics from Kibana: invalid response from Kibana status: 400 Bad Request
```

We can find those 400 in Kibana logs:

```
kubectl logs -n logging kibana-kube-5d57f6fd4c-64rpt -c kibana | grep ' 400 '
[...]
{"type":"response","@timestamp":"2020-11-22T17:24:54Z","tags":["api"],"pid":6,"method":"get","statusCode":400,"req":{"url":"/api/status?extended=","method":"get","headers":{"host":"localhost:5601","user-agent":"Go-http-client/1.1","accept":"application/json","accept-encoding":"gzip"},"remoteAddress":"127.0.0.1","userAgent":"Go-http-client/1.1"},"res":{"statusCode":400,"responseTime":56,"contentLength":9},"message":"GET /api/status?extended= 400 56ms - 9.0B"}
{"type":"response","@timestamp":"2020-11-22T17:25:24Z","tags":["api"],"pid":6,"method":"get","statusCode":400,"req":{"url":"/api/status?extended=","method":"get","headers":{"host":"localhost:5601","user-agent":"Go-http-client/1.1","accept":"application/json","accept-encoding":"gzip"},"remoteAddress":"127.0.0.1","userAgent":"Go-http-client/1.1"},"res":{"statusCode":400,"responseTime":99,"contentLength":9},"message":"GET /api/status?extended= 400 99ms - 9.0B"}
{"type":"response","@timestamp":"2020-11-22T17:25:54Z","tags":["api"],"pid":6,"method":"get","statusCode":400,"req":{"url":"/api/status?extended=","method":"get","headers":{"host":"localhost:5601","user-agent":"Go-http-client/1.1","accept":"application/json","accept-encoding":"gzip"},"remoteAddress":"127.0.0.1","userAgent":"Go-http-client/1.1"},"res":{"statusCode":400,"responseTime":75,"contentLength":9},"message":"GET /api/status?extended= 400 75ms - 9.0B"}
```

Is there something I should activate in Kibana?
Running from docker.elastic.co/kibana/kibana:7.10.0

Without the `?extended` param, the `/api/status` route returns with:

```
{"name":"kibana","uuid":"4d9a609d-75aa-4f66-b13b-5a44319c2260","version":{"number":"7.10.0","build_hash":"1796b5ec8fa1e60ccea63f2e5c25ccc665b92fdc","build_number":35949,"build_snapshot":false},"status":{"overall":{"since":"2020-11-22T17:22:07.688Z","state":"green","title":"Green","nickname":"Looking good","icon":"success","uiColor":"secondary"},"statuses":[{"id":"core:elasticsearch@7.10.0","message":"Elasticsearch is available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"core:savedObjects@7.10.0","message":"SavedObjects service has completed migrations and is available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:advancedSettings@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:bfetch@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:charts@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:devTools@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:discover@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:embeddable@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:esUiShared@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:expressions@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:indexPatternManagement@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:inspector@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:kibanaOverview@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:kibanaReact@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:kibanaUsageCollection@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:kibanaUtils@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:legacyExport@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:management@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:navigation@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:savedObjectsManagement@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:savedObjects@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:share@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:telemetryCollectionManager@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:telemetryManagementSection@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:uiActions@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:urlForwarding@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visDefaultEditor@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visualizations@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visualize@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:alerts@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:canvas@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:dashboardEnhanced@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:dataEnhanced@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:embeddableEnhanced@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:features@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:fileUpload@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:globalSearchBar@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:globalSearchProviders@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:grokdebugger@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:ingestPipelines@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:mapsLegacyLicensing@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:ml@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:painlessLab@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:searchprofiler@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:telemetryCollectionXpack@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:transform@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:translations@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:uiActionsEnhanced@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:uptime@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:watcher@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:xpackLegacy@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:urlDrilldown@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:apmOss@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:console@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:dashboard@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:data@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:home@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:inputControlVis@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:kibanaLegacy@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:mapsLegacy@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:newsfeed@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:regionMap@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:securityOss@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:telemetry@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:tileMap@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:timelion@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:usageCollection@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeMarkdown@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeMetric@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeTable@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeTagcloud@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeTimelion@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeTimeseries@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeVega@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeVislib@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:actions@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:apm@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:beatsManagement@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:case@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:cloud@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:code@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:consoleExtensions@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:crossClusterReplication@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:dashboardMode@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:discoverEnhanced@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:encryptedSavedObjects@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:enterpriseSearch@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:eventLog@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:globalSearch@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:graph@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:indexLifecycleManagement@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:indexManagement@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:infra@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:ingestManager@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:lens@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:licenseManagement@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:licensing@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:lists@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:logstash@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:maps@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:monitoring@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:observability@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:remoteClusters@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:reporting@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:rollup@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:securitySolution@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:snapshotRestore@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:spaces@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:stackAlerts@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:taskManager@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:triggersActionsUi@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:upgradeAssistant@7.10.0","message":"All dependencies are available","since":"2020-11-22T17:22:07.688Z","state":"green","icon":"success","uiColor":"secondary"}]},"metrics":{"last_updated":"2020-11-22T17:22:03.757Z","collection_interval_in_millis":5000,"os":{"platform":"linux","platformRelease":"linux-4.19.0-12-amd64","load":{"1m":0.64501953125,"5m":0.39599609375,"15m":0.26513671875},"memory":{"total_in_bytes":12296331264,"free_in_bytes":5623910400,"used_in_bytes":6672420864},"uptime_in_millis":24802000,"distro":"CentOS","distroRelease":"CentOS-8.2.2004","cpuacct":{"control_group":"/","usage_nanos":14084976608},"cpu":{"control_group":"/","cfs_period_micros":100000,"cfs_quota_micros":130000,"stat":{"number_of_elapsed_periods":606,"number_of_times_throttled":12,"time_throttled_nanos":274125707}}},"process":{"memory":{"heap":{"total_in_bytes":161497088,"used_in_bytes":142625224,"size_limit":1526909922},"resident_set_size_in_bytes":238989312},"pid":6,"event_loop_delay":1.1287220008671284,"uptime_in_millis":152557},"response_times":{"avg_in_millis":13,"max_in_millis":13},"concurrent_connections":0,"requests":{"disconnects":0,"total":1,"statusCodes":{"200":1},"status_codes":{"200":1}}}}
```

**Comment by mindw** [comment]
@faust64 This is most likely caused by https://github.com/elastic/kibana/pull/76054 .
The `?extended` param works with 7.9.x - looks like this API (which is undocumented) is being revised. https://github.com/elastic/kibana/issues/41983

Hope this helps!

**Comment by chamilad** [comment]
Thanks! Released with [`v7.10.x.1`](https://github.com/chamilad/kibana-prometheus-exporter/releases/tag/v7.10.x.1)
